db.go

```diff
@@ -687,9 +687,7 @@ func (db *DB) View(fn func(*Tx) error) error {
 		return err
 	}
 
-	if err := t.Rollback(); err != nil {
```

Comment:
I think the intent here is to avoid using unexported functions across object boundaries to avoid tight coupling.

It's probably be better to have:
```go
     return t.Rollback()
}
```

which would directly correspond with the `Update` code. `Rollback` is the method for closing read-only transactions; `close` is atypical.

Reply:
Make sense to me. `tx.close` is tx's own function. And ` return t.Rollback()` matches `return t.Commit()` in `Update` function.

I'll update this commit later.